chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -9,6 +9,537 @@
 #undef ParseCertificatesFile
 #undef ParseJSON
 
+namespace {
+base::StringPiece kBravePinsJson = R"brave_pins_json({
+  "pinsets": [
+    {
+      "name": "brave",
+      "static_spki_hashes": [
+        "AmazonRootCA1",
+        "AmazonRootCA2",
+        "AmazonRootCA3",
+        "AmazonRootCA4",
+        "StarfieldG2",
+        "GlobalSignRootCA_R1",
+        "GlobalSignRootCA_R3",
+        "GlobalSignRootCA_R6",
+        "GlobalSignRootCA_R46",
+        "GlobalSignRootCA_R5",
+        "GlobalSignRootCA_E46",
+        "DSTRootCA_X3",
+        "ISRGRootCA_X1",
+        "ISRGRootCA_X2"
+      ]
+    }
+  ],
+  "entries": [
+    // Critical endpoints that should remain unpinned so that they
+    // always work.
+    { "name": "laptopquery-updates.brave.com", "pins": "brave"},
```

Comment:
These ones have to be excluded from the pins (as the comment says). They should only be in the HSTS section.

Reply:
Great catch, fixed in https://github.com/brave/brave-core/pull/17033/commits/84b00b9a256cd4411301086df7f38e863a0cd082.